### PR TITLE
Added downstream namespace check for getting broker route

### PR DESCRIPTION
**Pull request by dymurray**

### CODE REVIEW

File: src/apb/engine.py [hunk 1]
```diff
@@ -394,6 +394,12 @@ def get_asb_route():
         openshift_config.load_kube_config()
         oapi = openshift_client.OapiApi()
         route_list = oapi.list_namespaced_route('ansible-service-broker')
+        if route_list.items == []:
+            print("Didn't find OAB route in namespace: ansible-service-broker. Trying openshift-ansible-service-broker")
```
Comment (rthallisey) [inline src/apb/engine.py]: What does OAB stand for?

File: src/apb/engine.py [hunk 2]
```diff
@@ -394,6 +394,12 @@ def get_asb_route():
...
+            route_list = oapi.list_namespaced_route('hift-ansible-service-broker')
```
Comment (rthallisey) [inline src/apb/engine.py]: ```hift-ansible-service-broker``` -> ```openshift-ansible-service-broker```

**Review by rthallisey — CHANGES_REQUESTED** [review]

**Review by cfchase — APPROVED** [review]
ACK

**Review by djzager — APPROVED** [review]
ACK

**Review by rthallisey — APPROVED** [review]

**Pull request merged**